index.js

```diff
@@ -0,0 +1,9 @@
+const calculateTotal = (items) => items.map(calculatedSubTotals)
+
+const calculatedSubTotals = (items) => {
```

Comment:
Remember map is calling this function on each object in the items array one at a time so `items` here is a misleading name. `item` would be better since this function is only ever passed a single object at a time.